apps/site/src/pages/[shortname]/types/entity-type/[...slug-maybe-version].page/entity-type-form.tsx

```diff
@@ -0,0 +1,249 @@
+import {
+  EntityType,
+  EntityTypeWithMetadata,
```

Comment:
Also, probably a problem with the [extract type editor PR](https://github.com/hashintel/hash/pull/2035) and not with this one, but when editing custom array property type's min and max values this error is being thrown:
```Uncaught ReferenceError: setImmediate is not defined```

Reply:
Thanks Luís – I'll have a look at this in the type editor, it will be a missing polyfill.

Reply:
Done in https://github.com/hashintel/hash/pull/2035/commits/87059b991f8d29efdd4b730a150a0d84e24709cf